src/tss2-sys/sysapi_util.c

```diff
@@ -304,3 +303,111 @@ int GetNumResponseHandles(TPM2_CC commandCode)
 {
     return GetNumHandles(commandCode, 0);
 }
+
+#ifdef DISABLE_WEAK_CRYPTO
+bool IsAlgorithWeak(TPM2_ALG_ID algorithm, TPM2_KEY_SIZE key_size)
+{
+    switch (algorithm) {
+        case TPM2_ALG_RSA:
+            if (key_size <= 2048)
```

Comment:
Shouldn't this be `key_size < 2048`? RSA-2048 is fine according to NIST to which CII refers.

Reply:
Correct, condition updated. Thanks